projects/packages/wp-js-data-sync/src/class-data-sync.php

```diff
@@ -170,19 +171,6 @@ public function _print_options_script_tag() {
 		wp_localize_script( $this->script_handle, $this->namespace, $data );
 	}
 
-	/**
-	 * Create a new instance of the Data_Sync class.
-	 *
-	 * @param $namespace     string - Each registry should have a unique name, typically plugin name, like `jetpack_boost`
-	 * @param $script_handle string - The script handle name to pass the variables to, typically the same as the plugin name,
-	 *                       but with a dash instead of underscore, like `jetpack-boost`
-	 * @param $plugin_page   string   - The slug name of the plugin page. If null, it will be assumed to be the same as the
-	 *                       registry name, formatted as a http parameter. `jetpack_boost` -> `jetpack-boost`
-	 * @param $parent_page   string   - The slug name for the parent menu (or the file name of a standard WordPress admin page).
-	 *                       Defaults to `admin`
-	 *
-	 * @return Data_Sync - A new instance of the Data_Sync class.
-	 */
```

Comment:
Odd it didn't require docs here?

Reply:
Phan never requires docs. As for phpcs, [it seems this project has disabled all the doc checks](https://github.com/Automattic/jetpack/blob/c6457f303ade601c4fbb648ecb1ce92f53653fa6/projects/packages/wp-js-data-sync/.phpcs.dir.xml#L34-L60).